tests/virtualization/universal/virtmanager_rm_devices.pm

```diff
@@ -62,12 +63,18 @@ sub run_test {
             detect_login_screen() if (!check_screen('virt-manager_viewer_disconnected', 5));
             close_guest();
         }
-        else {
-            record_soft_failure 'bsc#1221917 - [MU]Core Dump Occurs on Bare Metal SLES15 SP5/SP6 with Xen Following Disk Detachment in sles15sp5PV Guest Environment.' if ($guest =~ m/PV/i);
-        }
     }
-
     wait_screen_change { send_key 'ctrl-q'; };
+
+    # Workaround to return guests to initial sate, related to bsc#1221917
+    shutdown_guests();
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        if ($guest =~ m/PV/i) {
+            record_soft_failure 'bsc#1221917 - [MU]Core Dump Occurs on Bare Metal SLES15 SP4/SP5/SP6 with Xen Following Disk Detachment in PV Guest Environment.';
+            assert_script_run "virsh detach-disk $guest xvdb --config";
+        }
+    }
+    start_guests();
```

Comment:
Thank you for the fix, @tbaev .

I am a little bit confused now with the overall test in this virtmanager_rm_devices module. Maybe it is not caused by your PR... The bug is only effective for PV guest, right? Why do we stop test on both PV and HVM?
If this is indeed a problem but not by my misunderstanding, can you fix it together?

Reply:
I think I know why now. The original test only test PV guest. So after excluding it, nothing is tested.

Reply:
HVM testing in this module was excluded long time ago in this PR ( 2019 year ), not sure why there is no ticket. Maybe we can crate a ticket to research it a little bit more.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8783

Reply:
Thank you @tbaev . Please give the new ticket normal priority .